postings/views.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from postings.models import Posting, Comment
+from users.models    import User
+
+class PostingDeatilView(View):
+    # 여기는 데코레이터가...필요가 없나? 비회원도 포스팅 상세를 볼 수는 있으니까.. 필요없음?
+    def get(self, request, posting_id):
+        try:    
+            posting = Posting.objects.get(id = posting_id)
+            posting.hits += 1
+            posting.save()
+            
+            images = [{'image_id' : image.id, 'image_url' : image.image_url}
+                      for image in posting.image_set.all()]
+            user   = User.objects.get(id = posting.user_id)
```

Comment:
```suggestion
            user   = posting.user
```
user를 posting으로부터 찾으려는 시도 너무 좋았습니다.
그런데, 이미 posting 객체로부터 user을 알 수 있습니다.